### PR TITLE
Write Defenitions of Protobufs

### PR DESCRIPTION
**Pull request by leejiwon1125**
Closes Write gRPC proto #52

### CODE REVIEW

**Comment by betarixm** [comment]
Should be rebased as #60 merged

File: rpc/src/main/protobuf/master.proto [hunk 1]
```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package kr.ac.postech.paranode.rpc;
+
+service Master {
+    
+    rpc RegisterWorkerDirectory (RegisterRequest) returns (RegisterRelpy) {}
```
Comment (hataehyeok) [inline rpc/src/main/protobuf/master.proto:7]: chore: `RegisterRelpy` should fixed to `RegisterReply`

File: rpc/src/main/protobuf/master.proto [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+}
+
+message RegisterRequest {
+    string ipAddress = 1;
+    repeated string inputDirectory = 2;
+    string outputDirectory = 3;
+}
+
+message RegisterRelpy {
```
Comment (hataehyeok) [inline rpc/src/main/protobuf/master.proto:16]: its also changed to `RegisterReply`

**Review by hataehyeok — COMMENTED** [review]
I look over code now

File: rpc/src/main/protobuf/master.proto (hunk 2 above)
Comment (leejiwon1125) [reply rpc/src/main/protobuf/master.proto]: # https://github.com/betarixm/434project/pull/59/commits/f7b490679dd9d46949c4e52e45f3946ccaf3f8c1

File: rpc/src/main/protobuf/master.proto (hunk 1 above)
Comment (leejiwon1125) [reply rpc/src/main/protobuf/master.proto]: https://github.com/betarixm/434project/pull/59/commits/f7b490679dd9d46949c4e52e45f3946ccaf3f8c1

**Review by betarixm — APPROVED** [review]

**Review by hataehyeok — APPROVED** [review]

**Pull request merged**